### PR TITLE
[Keras] Add l2_normalize support

### PR DESCRIPTION
**Pull request by apivovarov**
This PR adds support for `Lambda(l2_normalize)`.
Keras does not have dedicated Layer for `l2_normalize`. [Typical solution](https://stackoverflow.com/questions/59596162/how-to-apply-l2-normalization-to-a-layer-in-keras) for that is to wrap `K.l2_normalize` with `Lambda` layer and add it to the model.

### CODE REVIEW

**Comment by apivovarov** [comment]
@AndrewZhaoLuo Could you review this PR?

**Comment by AndrewZhaoLuo** [comment]
I will review this either today or tomorrow

**Review by AndrewZhaoLuo — APPROVED** [review]
LGTM @masahi ?

File: .gitignore [hunk 1]
```diff
@@ -174,6 +174,7 @@ perf
 .bash_history
 *.json
 *.params
+*.ro
```
[the remark this answers is not in the document]
Comment (apivovarov) [reply .gitignore:177]: Relay vm exec code files
```
vm_exec = relay.vm.compile(mod, target=target, params=params)
code, lib = vm_exec.save()
with open("model.ro", "wb") as fo:
    fo.write(code)
```
This is not directly related to this PR. Just a minor fix

**Review by masahi — APPROVED** [review]

**Pull request merged**